### PR TITLE
BTV_HLT_RelVal - Fixing path names, adding 2d plots, input plots, fixing code checks

### PR DESCRIPTION
**Pull request by andrzejnovak**

### CODE REVIEW

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
Hi,

I don't see you in DQM contacts list, so could you please briefly introduce yourself?

**Comment by anovak10** [comment]
Hi, 

I guess I still wasn't added. I do the Release Validation for BTV HLT. @defranchis Can I be added to the appropriate twiki?

**Comment by defranchis** [comment]
hi @andrius-k, I am quite sure I already added @anovak10 a few weeks ago. Could you point me to a link or something to check? Thanks!

**Comment by andrius-k** [comment]
Hi @defranchis, sure, I'm checking this link: https://twiki.cern.ch/twiki/bin/viewauth/CMS/DQMContacts

Now I can see @anovak10 so everything is fine.

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
+1

**Comment by anovak10** [comment]
Added some more stuff

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
+1

File: HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc [hunk 1]
```diff
@@ -142,14 +216,29 @@ void HLTBTagPerformanceAnalyzer::analyze(const edm::Event& iEvent, const edm::Ev
...
 		    edm::LogInfo("NoCollection") << "Collection " << JetTagCollection_Label[ind] <<  " ==> not found"; return;
 		}
-
+		//fill Inputs for All
+		for(auto& info :  *(shallowTagInfosPf)) {
+			TaggingVariableList vars = info.taggingVariables();
+			for(auto entry = vars.begin(); entry != vars.end(); ++entry) 
+				{
+				if ( keepSet.find(TaggingVariableTokens[entry->first]) != keepSet.end()) {  // if Input name in defined list to keep
+					std::cout << entry->first  << TaggingVariableTokens[entry->first] <<  entry->second << std::endl;  //for debugging
```
Comment (kpedro88) [inline HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc]: remove cout

File: HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc [hunk 2]
```diff
@@ -211,12 +308,52 @@ void HLTBTagPerformanceAnalyzer::bookHistograms(DQMStore::IBooker & ibooker, edm
...
 		ibooker.setCurrentFolder(dqmFolder);
 		
 		//book 1D btag plot for 'all'
-		if ( JetTagCollection_Label[ind] != "" && JetTagCollection_Label[ind] != "NULL" ) { 
+		if ( !JetTagCollection_Label[ind].empty() && JetTagCollection_Label[ind] != "NULL" ) { 
 			H1_.back()[JetTagCollection_Label[ind]]       = ibooker.book1D(JetTagCollection_Label[ind] + "_all",      JetTagCollection_Label[ind]+ "_all",  btagBins, btagL, btagU );
 			H1_.back()[JetTagCollection_Label[ind]]      -> setAxisTitle(JetTagCollection_Label[ind] +"discriminant",1);
+			//Input storing 
+			ibooker.setCurrentFolder(dqmFolder+"/inputs");
+			/*for (int i=0; i < 100; i++) {
```
Comment (kpedro88) [inline HLTriggerOffline/Btag/src/HLTBTagPerformanceAnalyzer.cc]: delete commented-out code

**Comment by kpedro88** [comment]
please test

**Comment by anovak10** [comment]
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-24316/29985/runTheMatrix-results/4.22_RunCosmics2011A+RunCosmics2011A+RECOCOSD+ALCACOSD+SKIMCOSD+HARVESTDC/JobReport2.xml  
-- Looks like a problem with the input file, not the code

**Comment by boudoul** [comment]
Hi @anovak10 , indeed there is an issue with this file for all PR tests... but meanwhile would you mind to update the title of your PRs to make it a bit more explicit ? In the release notes we won't guess from the title  if this is for Tracking, Hcal, Ecal , or   HLT Btag ... thank you

**Comment by andrius-k** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Pull request merged**